cmd/podman/common.go

```diff
@@ -553,7 +558,18 @@ func getAuthFile(authfile string) string {
 	if authfile != "" {
 		return authfile
 	}
-	return os.Getenv("REGISTRY_AUTH_FILE")
+	if remote {
```

Comment:
do i read this correctly that you re-enable authfile for the remote client but basically no-op it?

Reply:
Just on the fence if we should do a logrus.debug or maybe info here to let them know we're in remote, auth is being ignored.

Reply:
Ditto certs.